sysboundary/sysboundary.cpp

```diff
@@ -677,18 +677,18 @@ void SysBoundary::applySysBoundaryVlasovConditions(
    for (uint popID = 0; popID < getObjectWrapper().particleSpecies.size(); ++popID) {
```

Comment:
update the comment on line 672.

Important note here is that lines 673-674 already update the important cell parameters and population metadata for extended sysb neighborhood, which is what L2 sysb cells now need. So good 👍